codechecker_common/checker_labels.py

```diff
@@ -125,34 +131,41 @@ def is_unique(labels: Iterable[str], label: str):
                             f'Label "severity" should be unique for checker '
                             '{checker}.')
 
-    def __get_analyzer_data(self, analyzers: Iterable[str]):
-        for analyzer, checkers in self.__data.items():
-            if analyzers is None or analyzer in analyzers:
-                yield analyzer, checkers
+    def __get_analyzer_data(self, analyzer: Optional[str] = None) -> Generator:
```

Comment:
Can you be please more specific for the type of the `Generator`: https://docs.python.org/3/library/typing.html#typing.Generator

e.g.: Generator[Tuple[str, Any]]